### DISCUSSION TITLE
Style does not work in ProLayout

### DISCUSSION THREAD

**Discussion by edikurniawan-dev** [typesnippet/antd-phone-input#93] [Q&A]
### Bug description

I'm using pro component, and i use prolayout for build my dashboard
if i use inside `<prolayout>` tag why the style not wrking, if i use out of `<prolayout>` tag style working very well

### Screenshots

inside tag prolayout
![image](https://github.com/typesnippet/antd-phone-input/assets/50087198/99064399-075b-4807-a36a-a871894fe8d1)

outside/not using prolayout
![image](https://github.com/typesnippet/antd-phone-input/assets/50087198/234fc3c4-f941-47c3-a003-d9633a435c9c)

### Browsers

Chrome

### OS

Windows

**Comment by edikurniawan-dev** [comment]
when i inspect element
![image](https://github.com/typesnippet/antd-phone-input/assets/50087198/f6a31ace-a519-478d-9764-6f1219b0ed19)

![image](https://github.com/typesnippet/antd-phone-input/assets/50087198/7594557c-1411-4294-aa7a-83e25d443b8d)

**Comment by ArtyomVancyan** [comment]
Hi @edikurniawan-dev, can you please share a link to the sandbox environment or a GitHub repository where I can reproduce and debug the issue?

**Comment by ArtyomVancyan** [comment]
Based on the provided screenshots, you have customized default Ant Design styles, which can be the cause of the issue. I have tried inside and outside of the `<ProLayout>` component, as you mentioned, and don't see any unexpected appearance or behavior.

**Comment by edikurniawan-dev** [comment]
I try in sandbox [https://codesandbox.io/p/sandbox/pensive-mclean-wlzs3z](https://codesandbox.io/p/sandbox/pensive-mclean-wlzs3z) and working perfectly like this
![image](https://github.com/typesnippet/antd-phone-input/assets/50087198/abc7a552-bb81-4536-b426-837d7d3afe99)


But i try in using laravel inertia with react in this repo [https://github.com/edikurniawan-dev/laravel-antd-pro-components](https://github.com/edikurniawan-dev/laravel-antd-pro-components)
the style not working
![image](https://github.com/typesnippet/antd-phone-input/assets/50087198/afcb8501-d5f2-41d6-abbe-d8085c03226b)

and when i try to disable `<ProConfigProvider>` the style working
![image](https://github.com/typesnippet/antd-phone-input/assets/50087198/4c170d67-7b1d-48e5-97af-1f1afc7b250e)

**Reply by ArtyomVancyan** [reply]
> But i try in using laravel inertia with react in this repo https://github.com/edikurniawan-dev/laravel-antd-pro-components
the style not working

This approves my assumption that the cause is not the `ProLayout` component. The problem may be with SSR. I cannot run the [provided](https://github.com/edikurniawan-dev/laravel-antd-pro-components) application, I suggest you adjust the styles in the [resources/css/app.css](https://github.com/edikurniawan-dev/laravel-antd-pro-components/blob/main/resources/css/app.css).
